packages/functional-tests/pages/layout.ts

```diff
@@ -19,6 +24,23 @@ export abstract class BaseLayout {
     return `${this.baseUrl}/${this.path}`;
   }
 
+  /**
+   * Checks that the current path maps to the POM's expected path. This can be called before querying for locators in
+   * child classes to make locators more robust and avoid false positives. If the current path does not exist in the URL,
+   * an error will be thrown.
+   */
+  checkPath() {
```

Comment:
@vpomerleau @Trinaa Since you all last had a look, I've moved this function up to BaseLayout from baseTokenCode. I did this for two reasons. 
 - It's closer path. 
 - It has proven useful at catching false positives in tests, so if we apply the pattern in this PR in other clean up, it'll be more accessible here.